Examples/Algorithms/TrackFitting/src/KalmanFitterFunction.cpp

```diff
@@ -129,14 +130,16 @@ ActsExamples::makeKalmanFitterFunction(
   cfg.resolvePassive = false;
   cfg.resolveMaterial = true;
   cfg.resolveSensitive = true;
-  Acts::Navigator navigator(cfg);
-  Propagator propagator(stepper, std::move(navigator));
-  Fitter trackFitter(std::move(propagator));
+  Acts::Navigator navigator(cfg); // logger.cloneWithSuffix("Navigator")
```

Comment:
my problem with this was that setting verbose will flood your logs with navigator/propagator/stepper output and that was not useful in my case

happy to add it back

Reply:
Isn't that the point though? If you don't want the output, don't set it to VERBOSE?

Reply:
in my case I only wanted the kalman fitter to give verbose output. everything else was a ton of noise

Reply:
Actually I would definitively propagate the VERBOSE through, because there are enough situations where the navigation log is the interesting one...
It's always possible to reduce down the output again with e.g. `./my_script.py | grep <logger name>`

Reply:
makes sense will do